actors/evm/src/lib.rs

```diff
@@ -51,12 +51,16 @@ fn test_method_selector() {
 #[repr(u64)]
 pub enum Method {
     Constructor = METHOD_CONSTRUCTOR,
-    // TODO: Do we want to use ExportedNums for all of these, per FRC-42?
     InvokeContract = 2,
     GetBytecode = 3,
     GetStorageAt = 4,
     InvokeContractDelegate = 5,
     GetBytecodeHash = 6,
+    // FRC-42 method exports
+    InvokeContractExported = frc42_dispatch::method_hash!("InvokeContract"),
+    InvokeContractDelegateExported = frc42_dispatch::method_hash!("InvokeContractDelegate"),
+    GetBytecodeExported = frc42_dispatch::method_hash!("GetBytecode"),
```

Comment:
This can _technically_ be exported, but we don't have a concrete need right now so I wouldn't.

Reply:
JRPC uses it https://github.com/filecoin-project/lotus/blob/03eee4311e64e1c3a94046d1bc2055b1e3d273bd/node/impl/full/eth.go#L236

Reply:
Off-chain knowing the specific actor type. FRC-0042 is designed for public "interfaces" that exist irrespective of the underlying code CID.